save-frontend/src/main/kotlin/com/saveourtool/save/frontend/externals/lodash/Debounce.kt

```diff
@@ -2,6 +2,8 @@
 
 package com.saveourtool.save.frontend.externals.lodash
 
+const val DEFAULT_DEBOUNCE_PERIOD = 500
```

Comment:
Let's maybe place it not in externals, but rather in some utils/constants?

Reply:
Oh I just put it as close to `debounce` as possible. Ok, will do it

Reply:
Fixed